packages/cli/src/ui/hooks/atCommandProcessor.ts

```diff
@@ -466,7 +469,8 @@ export async function handleAtCommand({
     toolCallDisplay = {
       callId: `client-read-${userMessageTimestamp}`,
       name: readManyFilesTool.displayName,
-      description: readManyFilesTool.getDescription(toolArgs),
+      description:
+        invocation?.getDescription() ?? 'Error creating tool invocation.',
```

Comment:
Users don't know or care what an invocation is. Make the description be something more user facing. E.g. "Error attempting to execute tool to read files"